zeppelin-server/src/main/java/org/apache/zeppelin/rest/NotebookRestApi.java

```diff
@@ -462,6 +462,30 @@ public Response deleteParagraph(@PathParam("noteId") String noteId,
   }
 
   /**
+   * Clear result of all paragraphs REST API
+   *
+   * @param noteId ID of Note
+   * @return JSON with status.ok
+   */
+  @POST
+  @Path("{noteId}/clear")
+  @ZeppelinApi
+  public Response clearAllParagraphOutput(@PathParam("noteId") String noteId)
+      throws IOException {
+    LOG.info("clear all paragraph output of note {}", noteId);
+
+    if (!noteId.isEmpty()) {
+      Note note = notebook.getNote(noteId);
+      if (note == null) {
+        return new JsonResponse(Status.NOT_FOUND, "note not found.").build();
+      }
+      note.clearAllParagraphOutput();
```

Comment:
yeah, i think @khalidhuseynov  is right here.
But after checking the code of notebookRestApi, there are no check here. we need to fix this...